pkg/protocol/snapshotcreator/snapshotcreator.go

```diff
@@ -120,13 +120,23 @@ func CreateSnapshot(opts ...options.Option[Options]) error {
 		return accumulator + details.Amount
 	}, iotago.BaseToken(0))
 
+	totalBasicOutputAmount := lo.Reduce(opt.BasicOutputs, func(accumulator iotago.BaseToken, details BasicOutputDetails) iotago.BaseToken {
+		return accumulator + details.Amount
+	}, iotago.BaseToken(0))
+
 	var genesisTransactionOutputs iotago.TxEssenceOutputs
-	genesisOutput, err := createGenesisOutput(opt.ProtocolParameters.TokenSupply()-totalAccountAmount, opt.GenesisSeed, engineInstance)
+	genesisOutput, err := createGenesisOutput(opt.ProtocolParameters.TokenSupply()-totalAccountAmount-totalBasicOutputAmount, opt.GenesisSeed, engineInstance)
 	if err != nil {
 		return ierrors.Wrap(err, "failed to create genesis outputs")
 	}
 	genesisTransactionOutputs = append(genesisTransactionOutputs, genesisOutput)
 
+	genesisBasicOutputs, err := createGenesisBasicOutputs(opt.BasicOutputs, engineInstance)
+	if err != nil {
+		return ierrors.Wrap(err, "failed to create genesis basic outputs")
+	}
+	genesisTransactionOutputs = append(genesisTransactionOutputs, genesisBasicOutputs...)
```

Comment:
add these after the accounts, because currently the testsuite has assumptions start index 1 to numAccounts are the account outputs. So if anyone adds a basic output in the normal snapshot suddenly the tests will fail. (I know, this is suboptimal, but I am just the messenger)

Reply:
done!